docs/make.jl

```diff
@@ -1,15 +1,19 @@
-using Documenter, StaticArrays
+using Documenter
+using StaticArrays
 
-makedocs(
-         modules = [StaticArrays],
-         sitename = "StaticArrays.jl",
-         pages = [
-            "Home" => "index.md",
-            "API" => "pages/api.md",
-            "Quick Start" => "pages/quickstart.md",
-            ],
-        )
-
-deploydocs(
-    repo = "github.com/JuliaArrays/StaticArrays.jl.git",
+makedocs(;
```

Comment:
I suggest this instead of all the individual `using StaticArrays`.
```suggestion
# Setup for doctests in docstrings
DocMeta.setdocmeta!(StaticArrays, :DocTestSetup, :(using StaticArrays))

makedocs(;
```

Reply:
Thanks, I didn't know the `DocMeta.setdocmeta!`.